### ISSUE TITLE
Text in center of donut chart should update to relfect total 

### ISSUE THREAD

**Issue by JohnPeng47** [IBM/carbon-charts#83]
Currently the center of the donut chart displays a random value specified in the configs. After updating, another random value is generated. We should display the something more meaningful, such as the sum of the slices, and can update this value when a slice is selected/deselected from the legend.

**Comment by theiliad** [comment]
This was a design decision by our team, since the center of the donut is not anticipated to always show a sum of values.

@dianatran18 or Nicole feel free to expand on that.

However, sum and other possible values could be configured as modes that will be configured by the developer when initializing the chart.

**Comment by theiliad** [comment]
I see, thanks @dianatran18. My mistake then

**Comment by theiliad** [comment]
So the design requirement is that the value shows the sum. Now you can also
still manually update values, although the sum also gets calculated
automatically, so we would need an additional feature in the future to stop
the calculation of sum if user is setting values manually.

TLDR; we still support both, although an optimization is needed to stop
calculating sum if manually updating center text.
